runtime/altair/src/migrations.rs

```diff
@@ -27,23 +27,24 @@ pub type UpgradeAltair1030 = (
 	asset_registry::CrossChainTransferabilityMigration,
 	orml_tokens_migration::CurrencyIdRefactorMigration,
 	pool_system::MigrateAUSDPools,
-	runtime_common::migrations::nuke::Migration<crate::Loans, crate::RocksDbWeight, 1>,
-	runtime_common::migrations::nuke::Migration<crate::InterestAccrual, crate::RocksDbWeight, 0>,
+	runtime_common::migrations::nuke::Migration<crate::Loans, RocksDbWeight, 1>,
+	runtime_common::migrations::nuke::Migration<crate::InterestAccrual, RocksDbWeight, 0>,
 	pallet_rewards::migrations::new_instance::FundExistentialDeposit<
 		crate::Runtime,
 		pallet_rewards::Instance2,
 		crate::NativeToken,
 		crate::ExistentialDeposit,
 	>,
+	asset_registry::RegisterLpEthUSDC<LiquidityPoolsPalletIndex, OrmlAssetRegistry, RocksDbWeight>,
 );
 
 /// The Upgrade set for Algol - it excludes the migrations already executed in
 /// the side releases that only landed on Algol (1028 & 1029) but not yet on
 /// Altair.
 #[cfg(feature = "testnet-runtime")]
 pub type UpgradeAltair1030 = (
-	runtime_common::migrations::nuke::Migration<crate::Loans, crate::RocksDbWeight, 1>,
-	runtime_common::migrations::nuke::Migration<crate::InterestAccrual, crate::RocksDbWeight, 0>,
+	runtime_common::migrations::nuke::Migration<crate::Loans, RocksDbWeight, 1>,
+	runtime_common::migrations::nuke::Migration<crate::InterestAccrual, RocksDbWeight, 0>,
```

Comment:
Note how we do NOT add the migration to the `Algol` Upgrade set since we have already registered the token there.

Reply:
Is this comment correctly associated with those lines?

Reply:
Yes and no. These changes are just about dropping the `crate::` reference prefix, but I left that note here since this block is the `Upgrade` migration set for Algol.

Reply:
Just for future readers:

- migration not added at all to Altair runtime